src/test/java/org/sqlite/StatementTest.java

```diff
@@ -421,6 +421,18 @@ public void dateTimeTest() throws SQLException {
         Date d = rs.getDate(1);
         assertEquals(day.getTime(), d.getTime());
     }
+    
+    @Test
+    public void defaultDateTimeTest() throws SQLException {
+        stat.executeUpdate("create table daywithdefaultdatetime (id integer, datetime datatime default current_timestamp)");
+        PreparedStatement prep = conn.prepareStatement("insert into daywithdefaultdatetime (id) values (?)");
+        prep.setInt(1, 1);
+        prep.executeUpdate();
+        ResultSet rs = stat.executeQuery("select * from day");
```

Comment:
the table is named `daywithdefaultdatetime`, not `day`. The error message (`SQL error or missing database (no such table: day)`) is here: https://github.com/xerial/sqlite-jdbc/runs/2936048257#step:5:4131

Reply:
```suggestion
        ResultSet rs = stat.executeQuery("select * from daywithdefaultdatetime");
```

Reply:
@bendem thank you!